src/components/GameCards/GameCard.js

```diff
@@ -79,6 +79,9 @@ export default {
       this.$emit('added');
       this.$store.commit('ADD_GAME', data);
 
+      // console.log(this);
+      // this parent parent refs = searchInput
+
```

Comment:
remove please

Reply:
See ed0aac2's commit message.

Basically I'm not sure how to focus the input in this case, since the ref is 2 parents above the `searchInput` ref.

Reply:
Ohh, you can emit a $bus event from here and have `AddGameModal.vue` listen for it...

you can look at `this.$bus.$emit('GAMES_ADDED');` for an example.

Reply:
Not entirely sure if that's a better approach but at least you don't have to pass the event twice

Reply:
Actually, you don't really have to focus back in the input, it's not the expected behavior, users know that clicking away from input will make it lose focus

Reply:
Alright, then I'll just remove the comments and hope the test doesn't fail again.